src/Translator/src/DependencyInjection/Configuration.php

```diff
@@ -29,6 +29,7 @@ public function getConfigTreeBuilder(): TreeBuilder
         $rootNode
             ->children()
                 ->scalarNode('dump_directory')->defaultValue('%kernel.project_dir%/var/translations')->end()
+                ->booleanNode('throw_when_not_found')->defaultTrue()->end()
```

Comment:
We don't want to introduce a parameter that can breaks existing apps, in a minor release:
```suggestion
                ->booleanNode('throw_when_not_found')->defaultFalse()->end()
```